tests/test_core.py

```diff
@@ -4582,20 +4599,43 @@ def process(filename):
       }
     ''', 'at Object.write', js_engines=[NODE_JS], post_build=post) # engines has different error stack format
 
+  def test_noderawfs(self):
```

Comment:
tests like these two that don't depend on the compiler mode should be in `other` (otherwise we run the same test in each mode)